packages/terra-list/src/List.jsx

```diff
@@ -61,12 +82,21 @@ const List = ({
   );
 
   const attrSpread = {};
+  attrSpread.role = 'list'; // Explicitly set role='list' as it's missing in Safari
   if (role && role.length > 0 && role !== 'none') {
     attrSpread.role = role;
   }
 
   return (
-    <ul {...customProps} {...attrSpread} className={listClassNames} ref={refCallback}>
+    <ul
+      aria-describedby={ariaDescribedBy}
+      aria-description={ariaDescription} // eslint-disable-line jsx-a11y/aria-props
+      aria-details={ariaDetails}
+      {...customProps}
```

Comment:
...customProps should be spread on top of other props to avoid unintended overridden of specified list props. ( if consumer has added both prop `ariaDescription`  and `aria-description` then value provided for prop `ariaDescription` will be over-ridden by customProps passed below it.

Reply:
[30c9afa](https://github.com/cerner/terra-core/pull/3688/commits/30c9afaafc79d12ece91550ddccda1f1f1bb671a)